Description: Use wrong compiler and pkgconfig file for configuration
	When cross compiling the source code, path to pkgconfig file
	need to consistent with host-specific

diff --git a/pal-0.4.3/debian/rules b/pal-0.4.3/debian/rules
index 1d8260c..e7decd1 100755
--- a/pal-0.4.3/debian/rules
+++ b/pal-0.4.3/debian/rules
@@ -24,7 +24,11 @@ build-indep:
 build-arch: build-stamp
 build-stamp: $(QUILT_STAMPFN)
 	dh_testdir
-	$(MAKE) "CFLAGS=$(CFLAGS)"
+	if [ "$(DEB_BUILD_GNU_TYPE)" != "$(DEB_HOST_GNU_TYPE)" ]; then \
+                PKG_CONFIG_PATH=/usr/lib/$(DEB_HOST_GNU_TYPE)/pkgconfig $(MAKE) CC="$(DEB_HOST_GNU_TYPE)-gcc" "CFLAGS=$(CFLAGS)"; \
+        else \
+                $(MAKE) "CFLAGS=$(CFLAGS)"; \
+        fi
 	touch $@
 
 .PHONY: clean
@@ -64,4 +68,3 @@ binary-arch: build install
 	dh_gencontrol
 	dh_md5sums
 	dh_builddeb
-
